content/docs/getting-started/overview/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Overview
+publishedDate: '2024-04-26T16:00:00.0Z'
+description: How to begin onboarding on to Roadie.
+---
+
+
+## Introduction
+
+Learn how to build your internal developer portal with Roadie:
+
+- Add users 
+- Populate your software catalog
+- Add plugins and customise your user interface
+- Dive into scorecards, templates and docs-as-code
+
+## 1. [Adding users](/docs/getting-started/user-management/)
+
+- [Invite](/docs/getting-started/user-management/) members of your team who will be responsible for Roadie. 
+- Designate [admins](/docs/getting-started/create-admin-group/) who will be responsible for configuring the Roadie system.
+- [Model users, groups and teams](/docs/getting-started/teams/) to build concepts of ownership into the catalog from the start.
+
+## 2. [Setting up your catalog](/docs/getting-started/adding-a-catalog-item/)
+
+We provide various ways to populate the catalog. Generally you will store entity files in a code repository and then provide Roadie with the URL of that file for it to be imported as a catalog entry. However you may also import a catalog item via the Roadie CLI or API.
+
+- **GitHub** - You can store a catalog item as yaml in a code repository on GitHub and import it into the Roadie catalog.
+- **Azure** - You can store a catalog item as yaml in a code repository in Azure and import it into the Roadie catalog.
```

Comment:
Do we want to be specific and say Azure DevOps when speaking specifically to SCM or keep it vague and always just say Azure?

Reply:
I can add DevOps